app/src/main/java/com/example/coctailapp/MainActivity.kt

```diff
@@ -14,21 +14,21 @@ import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
 import androidx.navigation.compose.NavHost
 import androidx.navigation.compose.composable
 import androidx.navigation.compose.rememberNavController
-import com.example.coctailapp.ui.screens.loginScreen
-import com.example.coctailapp.ui.screens.registerScreen
+import com.example.coctailapp.ui.screens.LoginScreen
+import com.example.coctailapp.ui.screens.RegisterScreen
 import com.example.coctailapp.ui.theme.CoctailAppTheme
 import kotlinx.serialization.Serializable
 
 class MainActivity : ComponentActivity() {
-    private var splash = true
+//    private var splash = false
```

Comment:
Please remove all commented code blocks and UNUSED imports in all files. ( Check Android Studio warnings for every file. We should have no warnings in any of the files)